plugins/mcreator-localization/help/default/item/condition_glow.md

```diff
@@ -1,2 +0,0 @@
-If condition is specified, the item will only glow in case this condition is passed.
-This will be ignored if glow effect is disabled.
```

Comment:
These help tips should also be removed from the other languages

Reply:
They are now automatically handled by Crowdin.

Reply:
Ah, Ok